moveit_kinematics/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ install(
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
+  INCLUDES DESTINATION include/moveit_kinematics
```

Comment:
Are you sure we need this line at all?

Reply:
I believe it can be removed. So long as another package in this repo depends on this and uses its headers, we can properly test this change. Because I'm not sure if that's true, I wasn't going to make that change.

Reply:
do we even need to install headers at all for all these plugins?

Reply:
I think it's not unreasonable to install plugin headers